tests/test_autoscaling/test_autoscaling.py

```diff
@@ -8,7 +8,8 @@
 import boto.ec2.elb
 import sure  # noqa
 
-from moto import mock_autoscaling, mock_ec2_deprecated, mock_elb_deprecated, mock_autoscaling_deprecated, mock_ec2
+from moto import mock_autoscaling, mock_ec2_deprecated, mock_elb_deprecated, mock_autoscaling_deprecated, \
```

Comment:
Let's revert the change for now purely because I don't think I have a sense for what the Moto style guidelines should be here. If you feel strongly one way or another please adjust the code accordingly.